.editorconfig

```diff
@@ -1,2 +1,2 @@
 [*.{gradle,mapping,unpick}]
-indent_style = tab
+indent_style = tab
```

Comment:
Don't know why these file ending changes happen, which doesn't quite mirror the current yarn head.

Reply:
So I should revert them?